Data Structures/Linked-Lists/forwardlist_using_stl.cpp

```diff
@@ -0,0 +1,102 @@
+#include<iostream>
+#include<vector>
+#include<forward_list>/*header file required for forward list implementation*/
+using namespace std;
+//display functions
+void displayint(forward_list <int> f)
```

Comment:
Follow [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention) everywhere in your code.